src/lib/djerba/plugins/fusion/plugin.py

```diff
@@ -67,22 +80,155 @@ def sort_by_actionable_level(row):
             }
             gene_info = []
             treatment_opts = []
+
+        # Processing fusions and generating blob URLs
+        tsv_file_path = wrapper.get_my_string(fc.ARRIBA_PATH)
+        base_dir = (directory_finder(self.log_level, self.log_path).get_base_dir())
+        fusion_dir = os.path.join(base_dir, "plugins", "fusion")
+        json_template_path = os.path.join(fusion_dir, fc.JSON_TO_BE_COMPRESSED)
+        output_dir = self.workspace.get_work_dir()
+        unique_fusions = list({item["fusion"] for item in results[fc.BODY]})
+        fusion_url_pairs = []
+
+        failed_fusions = 0
+
+        for fusion in unique_fusions:
+            try:
+                fusion, blurb_url = self.process_fusion(config, fusion, tsv_file_path, json_template_path, output_dir)
+                fusion_url_pairs.append([fusion, blurb_url])
+
+            except FusionProcessingError as e:
+                self.logger.error(f"Skipping fusion {fusion}: {e}")
```

Comment:
Use `self.logger.warning` -- error is for something which stops the program from completing